app/src/main/res/color/pager_dot.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/light_grey_50" android:state_selected="true" />
+    <item android:color="@color/photonLightGrey30" />
```

Comment:
do we want these to be attrs for light/dark ?

Reply:
This is a great catch, thank you!

Reply:
Fixed.

Reply:
Sorry I didn't mean just name the color . Did we want a different color for dark vs light? Like `?primaryText` Or did the UX mocks have `photonLightGrey30` for both light and dark.

Reply:
Should be fixed in the latest. Colors came over Slack - commented here https://github.com/mozilla-mobile/fenix/issues/8312#issuecomment-679292000.

Reply:
oh i can't read nvm lol this is ready!